spec/acceptance/suites/scl/redis5_spec.rb

```diff
@@ -4,11 +4,11 @@
 
 describe 'redis', if: %w[centos redhat].include?(os[:family]) && os[:release].to_i == 7 do
   before(:all) do
-    on hosts, puppet_resource('service', 'redis', 'ensure=stopped', 'enable=false')
+    apply_manifest('service{"redis" : ensure => stopped, enable => false}')
```

Comment:
A closer match (in multi host systems):
```suggestion
    apply_manifest_on(hosts, 'service{"redis" : ensure => stopped, enable => false}')
```

Or keeping it closer to the previous code I think this should still work:
```suggestion
    on hosts, puppet('resource', 'service', 'redis', 'ensure=stopped', 'enable=false')
```